test/selenium/util/TestUser.scala

```diff
@@ -0,0 +1,21 @@
+package selenium.util
+
+import java.time.Duration.ofDays
+
+import com.gu.identity.testing.usernames.TestUsernames
+
+class TestUser {
+
+  private val testUsers = TestUsernames(
+    com.gu.identity.testing.usernames.Encoder.withSecret(Config.testUsersSecret),
+    recency = ofDays(2)
+  )
+
+  private def addTestUserCookies(testUsername: String) = {
+    Driver.addCookie("ANALYTICS_OFF_KEY", "true")
```

Comment:
Here's the card: https://trello.com/c/t2gdI3y6/813-investigate-disabling-analytics-for-testing-purposes

Please add any extra details to this if you feel I haven't captured everything.